app/services/tagging/untagger.rb

```diff
@@ -0,0 +1,12 @@
+module Tagging
+  class Untagger
```

Comment:
Maybe CommonAncestorRemover (or CommonAncestorUntagger) would be a more informative name?

Reply:
It does general untagging - not necessarily to do with common ancestors

Reply:
Ok, that sounds fine.